lib/redis.ex

```diff
@@ -1,24 +1,45 @@
 defmodule Redbird.Redis do
-  def start_link(name) do
-    {:ok, client} = Exredis.start_link()
-    true = Process.register(client, name)
-    {:ok, client}
+  def child_spec(args) do
+    %{
+      id: Redbird.Redis,
+      start: {Redbird.Redis, :start_link, [args]}
+    }
   end
 
-  def get(value) do
-    Exredis.Api.get(pid(), value)
+  def start_link(opts) do
+    Redix.start_link(opts)
+  end
+
+  def get(key) do
+    pid()
+    |> Redix.command!(["GET", key])
+    |> case do
+      nil -> :undefined
```

Comment:
Where does `:undefined` come from? Should this still be the return value when the key is undefined?

Reply:
Great question. This is a value we're returning so that our `Plug.Session.REDIS` module knows to return the appropriate response for the session [here](https://github.com/thoughtbot/redbird/blob/master/lib/redbird/plug/session/redis.ex#L19).